CHANGELOG

```diff
@@ -1,3 +1,6 @@
+1.0.1:
+    - Add .npmignore
+    - Remove S3 publishing steps
```

Comment:
@jansepar do you think we'd just bump the version quietly then? Or not bump it at all?